### PR TITLE
 Rename knative-sandbox to knative-extensions

### PR DESCRIPTION
**Pull request by davidhadas**
This is an automated PR to replace  knative-sandbox strings with knative-extensions

### CODE REVIEW

File: .github/workflows/knative-profile-data-plane.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This file is automagically synced here from github.com/knative-sandbox/.github
+# This file is automagically synced here from github.com/knative-extensions/.github
```
Comment (davidhadas) [inline .github/workflows/knative-profile-data-plane.yaml:15]:
```suggestion
# This file is automagically synced here from github.com/knative-sandbox/.github
```

**Review by Cali0707 — APPROVED** [review]
/lgtm
/cc @creydr @matzew

**Review by creydr — COMMENTED** [review]
/approve

**Pull request merged**